### DISCUSSION TITLE
Global search input autofilled in Chrome

### DISCUSSION THREAD

**Discussion by invaders-xx** [filamentphp/filament#2133] [Help]
I create a new package for Jetstream profile within Filament and I have this strange behaviour where the global search field is filled with the email address : see below
<img width="1431" alt="image" src="https://user-images.githubusercontent.com/604907/162994413-192077ff-e3ba-464d-9852-e8eb611860a5.png">

@danharrin @ryangjchandler Any idea ?

**Comment by danharrin** [comment]
Its browser autofill, right?

**Reply by invaders-xx** [reply]
@danharrin only on this page ?

**Reply by danharrin** [reply]
Does it happen on other browsers?

**Reply by invaders-xx** [reply]
not on firefox, but on chrome yes

**Reply by danharrin** [reply]
Yeah, Chrome autofilling incorrectly is a very common problem. Please check - does the global search input have a `type="search"`?

**Reply by invaders-xx** [reply]
yes it has.

**Reply by danharrin** [reply]
And does your email input have `type="email"`?

**Reply by invaders-xx** [reply]
yes it has, but I can change it to text i think

**Reply by invaders-xx** [reply]
changing to text does nothing.

**Reply by danharrin** [reply]
not sure then mate, this is a Chrome bug and not Filament

**Reply by invaders-xx** [reply]
I realised that 👍 also. Anyway many thanks for your help.
